release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1609,6 +1609,8 @@ FATAL failed to fetch Metadata: failed to load asset "Install Config": platform.
 
 * Previously, the Azure clusters were created with the disk type of Premium_LRS and with an instance type that did not support PremiumIO capabilities which caused the cluster to fail. This update checks to see if the instance type picked has the PremiumIO capabilities only if the disk type is Premium_LRS which is the default disk type. The code queries the Azure subscription and region to get the information required and returns an error if the conditions are not met. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1931115[*BZ#1931115*])
 
+* Previously, the API VIP could become unavailable on the bootstrap when the API server restarts, which makes the provisioning services unavailable and provisioning fails. The provisioning services (Ironic) are now included in the VIP health checks, and the API VIP remains available. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949859[*BZ#1949859*])
```

Comment:
```suggestion
* Previously, the API VIP could become unavailable on the bootstrap when the API server restarted, which made the provisioning services unavailable and caused provisioning to fail. The provisioning services (Ironic) are now included in the VIP health checks, and the API VIP remains available. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949859[*BZ#1949859*])
```

Reply:
That's a great catch, I'm sorry I did not fix that previously. I will make that update. Thanks very much.